### PR TITLE
Added property removeImages to control image removal on composeDown

### PR DESCRIPTION
**Pull request by swalendzik**

### CODE REVIEW

**Comment by mzielinski** [comment]
+1

File: src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy [hunk 1]
```diff
@@ -22,8 +22,12 @@ class ComposeDown extends DefaultTask {
             }
             if (extension.removeContainers) {
                 if (getDockerComposeVersion() >= VersionNumber.parse('1.6.0')) {
+                    String[] args = ['down']
+                    if(extension.removeImages) {
+                        args += ['--rmi', 'all', '--volumes']
```
Comment (augi) [inline src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeDown.groovy]: Please add `--volumes` options as default (so have `String[] args = ['down', '--volumes']`). Or add also property for volumes removal. Thanks!

**Comment by aschrijver** [comment]
I wonder whether the `removeImages` should default to `true`. Containers is one thing, but wouldn't it be better to have removing images default to `false`?

**Comment by augi** [comment]
We had discussion about this and set default to `true` because it's help to prevent increasing disk usage on integration servers (every `up` creates new images in our use-case). But it's questionable...

**Comment by mzielinski** [comment]
When can we expect release with this improvement?

I think also, that we should set default value to true. Thanks to that, we won't break backward compatibility.

**Comment by aschrijver** [comment]
Yeah, the images can accumulate quickly. But it the flag is there you can configure to remove them. With regards to my tests, most of them do not test the creation of the image, just need to spin up containers quickly, run tests, report, done. From a testing perspective `false` makes more sense as a default IMHO

I certainly prefer a default = `false`. 

PS. I am also using the `ComposeUp` / `ComposeDown` tasks separately (in a number of `deployXYZBuild`, `undeployXYZBuild` tasks) where images should not be removed as well.

**Comment by aschrijver** [comment]
With regards to backward-compatibility. The images used to be not removed, am I right?

**Comment by augi** [comment]
@mzielinski sorry, I missed your commits :( Released as `0.2.98`.

**Pull request merged**